client/src/components/Home/index.js

```diff
@@ -249,11 +249,11 @@ export default function Home(props) {
               <p>
                 <a href="#" className="header-link">
                   {t("unsigned-home-demo.user-guides")}
-                </a>{" "}
+                </a>{"https://docs.rocket.chat/guides/user-guides"}
                 |{" "}
                 <a href="#" className="header-link">
```

Comment:
Same for this one. We need to replace the `#` by `https://github.com/RocketChat/Rocket.Chat/releases`.